apps/studio.giselles.ai/middleware.ts

```diff
@@ -20,8 +20,11 @@ export default supabaseMiddleware(async (user, request) => {
 	}
 });
 
-export const config = {
+export const config: MiddlewareConfig = {
 	matcher: [
-		"/((?!_next/static|_next/image|.well-known|dev|webhooks|legal|login|signup|join|pricing|password_reset|subscription|auth|api/giselle|api/vector-stores|favicon.ico|.*\\.(?:svg|png|jpg|jpeg|gif|webp)$).*)",
+		"/apps/:path*",
+		"/settings/:path*",
+		"/workspaces/:path*",
+		"/api/giselle/:path*",
 	],
```

Comment:
I think these routes would be better handled with middleware, too.
While there's no immediate issue with accessing these routes now, I believe they're intended to be accessed after login.

```suggestion
		"/connected",
		"/github/:path*",
		"/subscriptions/:path*"
	],
```